README.md

```diff
@@ -14,30 +14,41 @@ A [Singer](https://singer.io/) postgres target, for use with Singer streams gene
 
 ## Usage
 
-Create a config file with postgres connection information and target postgres schema.
-
-```json
-{
-  "postgres_host": "locahost",
-  "postgres_port": 5432,
-  "postgres_database": "my_analytics",
-  "postgres_username": "myuser",
-  "postgres_password": "1234",
-  "postgres_schema": "mytapname"
-}
-```
-
-Run `target-postgres` against a [Singer](https://singer.io) stream.
-
-```sh
-	tap-something | target-postgres --config config.json
-```
-
-It ignores "STATE" type Singer messages.
+1. Follow the
+   [Singer.io Best Practices](https://github.com/singer-io/getting-started/blob/master/docs/RUNNING_AND_DEVELOPING.md#running-a-singer-tap-with-a-singer-target)
+   for setting up separate `tap` and `target` virtualenvs to avoid version
+   conflicts.
+
+1. Create a config file at `/tmp/target_config.json` with postgres connection
```

Comment:
Does maybe `~/target_config.json` make more sense for this example? Why do you want to have the config be temporary? Most people trying this out will just be running it on their laptop and want to keep the config around.